serverless/functions/serverless-functions-getting-started.adoc

```diff
@@ -6,11 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+This guide explains how you can get started with creating, building, and deploying a function on an {ServerlessProductName} installation. When building and deploying functions, the resulting container image is stored in an image registry.
```

Comment:
IBM SG wants us to avoid self referential language like "this guide." Can you simply say?:

`Get started with creating, building, and deploying a function on an {prod} installation.`

Reply:
This text wasn't added in this PR (just moved), and there is a follow up Jira related to this to fix the abstracts for all the Functions docs and make them IBM SG and Jupiter compliant (https://issues.redhat.com/browse/SRVCOM-1831) so I'll leave it as is for now. "Get started with creating, building, and deploying a function on an {prod} installation." doesn't really read properly to me.